packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2843,7 +2845,7 @@ export class ContainerRuntime
 			deserializedContent: JSON.parse(serializedContent), // Deep copy in case caller changes reference object
 			metadata,
 			localOpMetadata,
-			referenceSequenceNumber: this.deltaManager.lastSequenceNumber,
+			referenceSequenceNumber: this.lastProcessedSequenceNumber,
```

Comment:
I do not think having this.lastProcessedSequenceNumber buys us anything. Old code does as good.
What is important (in this PR) is that we pass this info to Loader layer - we were already capturing right data here, just not passing it through.